packages/headless/src/features/search/search-mappings.ts

```diff
@@ -37,19 +37,23 @@ function mapDateRangeRequest(
   facetId: string,
   mappings: SearchMappings
 ) {
-  let start = value.start;
-  let end = value.end;
-  if (isRelativeDateFormat(start)) {
-    start = formatRelativeDateForSearchApi(start);
-    mappings.dateFacetValueMap[facetId][formatStartFacetValue(start)] =
-      value.start;
-  }
-  if (isRelativeDateFormat(end)) {
-    end = formatRelativeDateForSearchApi(end);
-    mappings.dateFacetValueMap[facetId][formatEndFacetValue(end)] = value.end;
-  }
+  try {
+    let start = value.start;
+    let end = value.end;
+    if (isRelativeDateFormat(start)) {
+      start = formatRelativeDateForSearchApi(start);
+      mappings.dateFacetValueMap[facetId][formatStartFacetValue(start)] =
+        value.start;
+    }
+    if (isRelativeDateFormat(end)) {
+      end = formatRelativeDateForSearchApi(end);
+      mappings.dateFacetValueMap[facetId][formatEndFacetValue(end)] = value.end;
+    }
 
-  return {...value, start, end};
+    return {...value, start, end};
+  } catch (error) {
+    return null;
```

Comment:
Should we log a warning here?

Reply:
If we are covering the entry points, then we would not need try/catch at this stage,